java/mtsj/core/src/main/java/com/devonfw/application/mtsj/general/common/base/JWTLoginFilter.java

```diff
@@ -22,24 +25,30 @@
  */
 public class JWTLoginFilter extends AbstractAuthenticationProcessingFilter {
 
+  private UserDetailsService userDetailsService;
+
   /**
    * The constructor.
    *
    * @param url the login url
    * @param authManager the {@link AuthenticationManager}
    */
-  public JWTLoginFilter(String url, AuthenticationManager authManager) {
+  @Autowired
```

Comment:
Usually we discourage constructor injection and also use JEE standards (`@Inject`) instead of proprietary spring annotations (`@Autowirde`) according to devon4j guidelines. Are there specific needs that force us to do this?

Reply:
Currently it already has url and authManager in constructor, tried @inject for userDetailservice but getting null, it wasn't able to inject it. I will create an issue for this to remove constructor based injection